ktlint-ruleset-experimental/src/main/kotlin/com/pinterest/ktlint/ruleset/experimental/ArgumentListWrappingRule.kt

```diff
@@ -139,18 +139,19 @@ class ArgumentListWrappingRule : Rule("argument-list-wrapping") {
                                     if (childIndent == intendedIndent) {
                                         continue@nextChild
                                     }
-                                    emit(
-                                        child.startOffset,
-                                        "Unexpected indentation" +
-                                            " (expected ${intendedIndent.length - 1}, actual ${childIndent.length - 1})",
-                                        true
-                                    )
+                                    // Do not emit a warning about incorrect indentation. Final indentation is
+                                    // determined by the indent rule. Displaying, possibly conflicting warnings about
+                                    // the exact indenting position is really confusing for the user.
                                 } else {
                                     emit(child.startOffset, errorMessage(child), true)
                                 }
                                 if (autoCorrect) {
-                                    val finalIndent =
-                                        (if (cut > -1) spacing.substring(0, cut) else "") + intendedIndent
+                                    // Although no warning is displayed about incorrect indentation, an attempt is
```

Comment:
This seems strange that we wouldn't emit a warning on linting, but we'd change indentation on formatting.

Reply:
Yes it is a bit unusual. But look at the number of issue we get about those warnings. So the warning has to be removed at minimum. Keeping the indent fix during format only has value in case the indent rule is disabled.

Reply:
We shouldn't have different behavior for linting vs formatting.

Reply:
I am fine with removing the functionality to fix indentation inside the argument/parameter list wrapping rule in order to make linting and formatting consistent again. Some users might consider this a breaking change.

@shashachu please confirm whether that is the way to go.

Reply:
yes, let's disable autocorrection of indentation for now